recipes/asap/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set ASAP_VERSION = "1.6" %}
+package:
+    name: asap
+    version: {{ ASAP_VERSION }}
+
+source:
+    fn: {{ ASAP_VERSION }}.zip
+    url: https://github.com/GeertLitjens/ASAP/archive/{{ ASAP_VERSION }}.zip
+
+build:
+    number: 0
+    skip: True  # [py3k or osx]
+
+requirements:
+    build:
+        - gcc   # [linux]
+        - cmake
+        - python
+        - boost >=1.55
+        - opencv >=2.4
+        - qt >=5.1
+        - libtiff >=4.0
+        - jpeg
+        - jasper
+        - dcmtk
+        - swig
+        - openslide
+        - pugixml
+        - zlib
+
+    run:
+        - python
```

Comment:
libgcc and these guidelines: https://bioconda.github.io/guidelines.html#c-c
there is also a section about boost and here an example: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/salmon/meta.yaml#L25